lib/generators/react_on_rails/base_generator.rb

```diff
@@ -62,7 +62,7 @@ def update_git_ignore
           /app/assets/javascripts/generated/*
         DATA
 
-        dest_file_exists?(".gitignore") ? append_to_file(".gitignore", data) : puts_setup_file_error(".gitignore", data)
+        dest_file_exists?(".gitignore") ? append_to_file(".gitignore", data) : options.errors.push(return_setup_file_error(".gitignore", data))
```

Comment:
@justin808 I agree, it does seem hacky. I'll explore your suggestion. Thanks for the feedback!